modules/developers-guide/examples/access-hello/main.mo

```diff
@@ -4,7 +4,7 @@ import Error "mo:base/Error";
 import List "mo:base/List";
 import Prim "mo:prim";
 
-actor {
+shared { caller = initializer } actor class() {
 
     // Establish role-based greetings to display
     public shared { caller } func greet(name : Text) : async Text {
```

Comment:
```suggestion
    public shared(msg) func greet(name : Text) : async Text {
```
Andreas wants you to rewrite like so here.

Reply:
(and similarly for other shared functions accessing the caller in this file)

Reply:
The preference for `shared(msg)` seems fine as a standard style preference.

Shall we add it to a special file (somewhere?) where we remember these choices for the future?

The reason I ask is that there's always a temptation to make code more compact (binding/projecting just `{caller}` instead of binding `msg` plus projecting `msg.caller`).  In the absence of preferences that ask for the longer style, we may regress here or elsewhere in some future PR.

Reply:
Sounds like this should go into the style code (though I find it clunky myself)

Reply:
I think the examples in the motoko documentation do adhere to this style now (unless some got away)